src/parser/warrior/arms/CHANGELOG.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 
-import { Aelexe, Zerotorescue, Sharrq } from 'CONTRIBUTORS';
+
+import { Aelexe, Zerotorescue, Sharrq, Matardarix } from 'CONTRIBUTORS';
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 
 export default [
+  {
```

Comment:
you might consider using a `<ul>` here to make it easier to read. not necessary, just a thought

Reply:
Looks like this now
![image](https://user-images.githubusercontent.com/10074100/47992692-d1692500-e0ed-11e8-9cdb-02acb43ca191.png)